code/datums/components/omen.dm

```diff
@@ -78,7 +78,7 @@
 			return
 
 		for(var/obj/machinery/vending/darth_vendor in the_turf)
-			if(darth_vendor.tiltable)
+			if(darth_vendor.tiltable && prob(50)) // SKYRAT EDIT CURSED QUIRK NERF - OLD: 	if(darth_vendor.tiltable)
```

Comment:
Undocumented nerf to people getting bad omen through normal means (like the FUCK YOU coupon)

Reply:
yeah tbf i could write that better so it doesn't touch original omen